docs/en/rst/using/creating-an-account.rst

```diff
@@ -7,7 +7,7 @@ If you want to use a particular installation of Bugzilla, first you need to
 create an account. Ask the administrator responsible for your installation
 for the URL you should use to access it. If you're test-driving Bugzilla,
 you can use one of the installations on
-`Landfill <http://landfill.bugzilla.org/>`_.
+`Landfill <https://bugzilla-dev.allizom.org/>`_.
```

Comment:
We should probably not refer to the new testing site as Landfill, but I wasn't sure what to call the new site. Any guidance here would be greatly appreciated!